src/Reverse.Proxy/nginx.conf

```diff
@@ -0,0 +1,19 @@
+user nginx;
+
+events {
+    worker_connections 1000;
+}
+http {
+
+  server {
+    listen 4000;
```

Comment:
Since it is running within a container and you can remap the port, I guess 80 would be fine.

Reply:
Ok Will do.